source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,454 @@
+* VMware image: `clear-<version>-basic.vmdk.xz` (for Method 2)
+
+For older versions, see the `releases`_ page.
+
+Although not required, it is recommended to download the corresponding 
+checksum file (designated with `-SHA512SUMS` at the end of the filename) 
+for the image in order to verify its integrity.
+
+Verify the integrity of the download (recommended)
+==================================================
+
+* For Linux distros and macOS:
+
+  #.  Start a terminal emulator.
+  #.  Go to the directory with the downloaded files.
+  #.  To verify the integrity of the image, enter the following (an installer ISO
+      image is used as an example):
+
+      .. code-block:: console
+
+        $ sha512sum ./clear-<version>-installer.iso.xz | diff ./clear-<version>-installer.iso.xz-SHA512SUMS -
+
+      If the checksum of the downloaded image is different than the original
+      checksum, the differences will displayed. An empty output indicates a match.
+
+* For Windows:
+
+  #.  Start Command-Prompt.
+  #.  Go to the directory with the downloaded files.
+  #.  To verify the integrity of the image, enter the following commands:
 
       .. code-block:: console
 
-         $ xz -d clear-{release-number}-live.img.xz
+        C:\> CertUtil -hashfile ./clear-<version>-installer.iso.xz sha512
+
+      Manually compare the output with the original checksum value shown in 
+      the downloaded checksum file (i.e. `clear-<version>-installer.iso.xz-SHA512SUMS`) 
+      to make sure they match.
+
+Uncompress the image
+====================
+
+* For Linux distros (an installer ISO image is used as an example):
+
+  .. code-block:: console
+
+    $ unxz clear-<version>-installer.iso.xz
+
+* For macOS:
+
+  .. code-block:: console
+
+    $ gunzip clear-<version>-installer.iso.xz
+
+* For Windows:
+
+  Use `7zip`_ to uncompress it.
+
+Method 1: Install Clear Linux into a new VM 
+===========================================
+
+The general process for performing a fresh install of |CL| into a new VM is 
+as follows (with expanded details below):
+
+* Upload the |CL| installer ISO to the VMware server
+* Create a new VM and configure its settings
+* Attach the installer ISO to it
+* Install |CL|
+* Detach the installer ISO
+* Enable UEFI boot support
+* Power on the VM
+
+Upload the Clear Linux installer ISO to the VMware server
+*********************************************************
+
+#.  Connect to the VMware server and log into an account with root privilege.
+#.  Under the :guilabel:`Navigator` window, go to :guilabel:`Storage`.
+#.  Click :guilabel:`Datastore browser`.
+    
+    |vmware-esxi-01|
+
+    Figure 1: VMware ESXi - Navigator > Storage 
+
+#.  Click :guilabel:`Create directory` and name it `ISOs`.
+
+    |vmware-esxi-02|
+
+    Figure 2: VMware ESXi - Datastore > Create directory 
+   
+#.  Select the newly created directory and click :guilabel:`Upload`.
+
+    |vmware-esxi-03|
```

Comment:
Do not use substitutions for figures.  Use the figure directive: http://docutils.sourceforge.net/docs/ref/rst/directives.html#figure
I can enable numbered references for Sphinx to autonumber figures with captions. IMO this could be very useful for tutorials with multiple screenshots. What do you think?

Reply:
@rcaballeromx #61 is for the Desktop version (Workstation Player) of VMware while #62 is for the cloud version (ESXi) of VMware.

Reply:
@rcaballeromx - I can change to "inline" figures.  But I am only following examples from previous docs that used substitutions.  Would be nice to have a "Bible" of what are our standards so that we're on the same page.